src/devices/machine/hd64610.cpp

```diff
@@ -228,7 +228,7 @@ void hd64610_device::rtc_clock_updated(int year, int month, int day, int day_of_
 	write_counter(REG_DAY, day);
 	write_counter(REG_MONTH, month);
 	write_counter(REG_YEAR, year);
-	m_regs[REG_DAY_OF_THE_WEEK] = day_of_week;
+	m_regs[REG_DAY_OF_THE_WEEK] = day_of_week - 1;
```

Comment:
I’ve applied this in 5d7d18a9c6e999aaed0d358436bd947901875053.  Kind of annoying that `device_rtc_interface` doesn’t use the conventional 0-based format for this field.